<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WithdrawInfo</class>
 <widget class="QDialog" name="WithdrawInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="messageLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>531</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="withdrawOkButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>460</y>
     <width>130</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="inventoryBrowser">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>60</y>
     <width>491</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
